style.css

```diff
@@ -1 +1,464 @@
+h5,
+h6,
+p,
+ol,
+ul {
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*----------  Header  ----------*/
+
+.header {
+  background-color: rgba(0, 0, 0, 0.05);
+  padding: 0 1rem;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.brand {
+  display: flex;
+  justify-content: space-between;
+}
+
+.brand-img {
+  max-width: 60px;
+  max-height: 60px;
+  margin: 1rem 0;
+}
+
+.brand-heading {
+  font-size: 2.6rem;
+  display: flex;
+  align-self: center;
+}
+
+.nav {
+  display: none;
+}
+
+.nav-list {
+  list-style: none;
+  width: 100%;
+}
+
+.nav-list-item {
+  text-align: center;
+  margin: 1rem 0;
+}
+
+.nav-link {
+  width: 100%;
+  display: inline-block;
+  font-size: 2rem;
+}
+
+.nav-link:hover {
+  background-color: pink;
+}
+
+.hamburger {
+  align-self: center;
+  user-select: none;
+  cursor: pointer;
+}
+
+.menu-visible {
+  display: flex;
+}
+
+/*----------  Hero section  ----------*/
+
+.hero {
+  min-height: 30rem;
+  background-image: url(images/cakes/cupcakes.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.hero-content {
+  background: rgba(200, 200, 200, 0.7);
+  padding: 1rem 2rem;
+  margin: 3rem 0;
+}
+
+
+.hero-content>p {
+  text-align: center;
+  color: rgba(0, 0, 0, 0.7)
+}
+
+.hero-content>h2 {
+  font-size: 3rem;
+  text-align: center;
+}
+
+/*----------  Carousel section  ----------*/
+
+.cakes-heading {
+  font-size: 3rem;
+  text-align: center;
+}
+
+.carousel-image {
+  width: 100%;
+  max-height: 260px;
+  object-fit: cover;
+  margin: 0 auto;
+  border-radius: 5px;
+}
+
+.carousel-item {
+  display: none;
+}
+
+.visible {
+  display: block;
+}
+
+.prev,
+.next {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  cursor: pointer;
+  font-size: 5rem;
+  color: #fff;
+  font-weight: bold;
+}
+
+.prev:hover,
+.next:hover {
+  font-size: 6rem;
+}
+
+.prev {
+  padding: 8rem 8rem 8rem 5rem;
+  left: 0;
+}
+
+.next {
+  padding: 8rem 5rem 8rem 8rem;
+  right: 0;
+}
+
+.carousel-wrapper {
+  max-width: 960px;
+  user-select: none;
+  position: relative;
+  margin: 0 auto;
+}
+
+.cakes-heading {
+  margin: 4.8rem 0;
+  transform: skewY(-1deg);
+}
+
+.cakes-heading>span {
+  background: pink;
+  padding: 1rem 2rem;
+}
+
+.info-wrapper {
+  margin: 0 1rem;
+}
+
+.info-heading {
+  font-size: 2.6rem;
+  margin: 2.8rem 0;
+  text-align: center;
+  letter-spacing: 2px;
+}
+
+.info-content {
+  font-size: 1.8rem;
+  max-width: 60rem;
+  margin: 0 auto;
+  letter-spacing: 2px;
+}
+
+/*----------  Cakes examples section  ----------*/
+
+.example-cakes {
+  background: linear-gradient(#fff, #e6e6e6);
+  padding: 3rem 0 10rem 0;
+}
+
+.example-cakes-wrapper {
+  margin-top: 7rem;
+}
+
+.example-image {
+  max-width: 100%;
+  display: block;
+  margin: 2rem auto;
+  border-radius: 6px;
+  box-shadow: 0px 6px 6px 0px rgba(0, 0, 0, 0.4)
```

Comment:
Because of the hover scale effect, you could add a transition here to make the change less abrupt.